web/src/rendered_markdown.ts

```diff
@@ -65,7 +67,7 @@ function get_user_group_id_for_mention_button(elem) {
     return undefined;
```

Comment:
It might be a sensible follow-up to make this assert that it got a value and never return `undefined`; unlike the `get_user_id_for_mention_button` code path, which has a specific story for how it might fail to find something ... I think it'd reflect a bugged server if there were ever one of these elements without a `data-user-group-id`.

Reply:
I did this as a commit appended to the end of this PR